CriticalMass/NextRideManager.swift

```diff
@@ -40,7 +40,9 @@ class NextRideManager {
     }
 
     private func getUpcomingRide(_ rides: [Ride]) -> Ride? {
-        rides.first { $0.dateTime > .now }
+        rides
+            .sorted(by: \.dateTime)
+            .first { $0.dateTime > .now }
```

Comment:
A small improvement could be using a lazy colelction here 

```suggestion
        rides
            .lazy      
            .sorted(by: \.dateTime)
            .first { $0.dateTime > .now }
```